modules/indexing/src/main/java/org/apache/ignite/internal/processors/query/stat/ColumnStatistics.java

```diff
@@ -60,13 +64,38 @@
      * @param raw Raw data to aggregate statistics.
      */
     public ColumnStatistics(Value min, Value max, int nulls, int cardinality, long total, int size, byte[] raw) {
+        this(min, max, nulls, cardinality, total, size, raw, 0);
```

Comment:
Why do we need special methods for test only? Let's add an additional parameter to test classes instead.